package.json

```diff
@@ -4,6 +4,14 @@
   "description": "",
   "author": "Aleix Casanovas (aleics)",
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/aleics/ngx-propserve.git"
```

Comment:
I think `https://github.com/aleics/ngx-propserve.git` should be sufficient, right? No need to define here any commit hash, or other git related stuff, so `git+` shouldn't bee needed.